aioshelly/rpc_device/device.py

```diff
@@ -206,11 +217,12 @@ async def initialize(self) -> None:
             self._last_error = DeviceConnectionError(err)
             _LOGGER.debug("host %s:%s: error: %r", ip, port, self._last_error)
             raise self._last_error from err
-        finally:
-            self._initializing = False
-
-        if self._update_listener and self.initialized:
-            self._update_listener(self, RpcUpdateType.INITIALIZED)
+        else:
+            _LOGGER.debug("host %s:%s: RPC device init finished", ip, port)
+            # Set notification callback once we are finished
+            # the init process
+            self._wsrpc.set_notification_callback(self._on_notification)
```

Comment:
We don't accept notifications until after https://github.com/home-assistant-libs/aioshelly/pull/618/files#diff-b3a38f3aae5db155a39278d8fad4bd7b03c9255b5fa3dff2ae6d072fccf83b7eR207 is done.

I think thats ok, but someone should double check that makes sense